integrations/aws/main.py

```diff
@@ -33,21 +37,51 @@
 )
 
 
+async def _handle_global_resource_resync(
+    kind: str,
+    credentials: AwsCredentials,
+) -> ASYNC_GENERATOR_RESYNC_TYPE:
+    denied_access_to_default_region = False
+    default_region = get_default_region_from_credentials(credentials)
+    default_session = await credentials.create_session(default_region)
+    try:
+        async for batch in resync_cloudcontrol(kind, default_session):
+            yield batch
+    except Exception as e:
+        if is_access_denied_exception(e):
+            denied_access_to_default_region = True
+        else:
+            raise e
+
+    if denied_access_to_default_region:
+        logger.info(f"Trying to resync {kind} in all regions until success")
+        async for session in credentials.create_session_for_each_region():
+            try:
+                async for batch in resync_cloudcontrol(kind, session):
+                    yield batch
+                break
+            except Exception as e:
+                if not is_access_denied_exception(e):
+                    raise e
+
```

Comment:
lets log if its access denied

Reply:
it will log it twice since it's already logging it in the `resync_cloudcontrol` function